src/common.ts

```diff
@@ -25,3 +25,12 @@ export function getMinBlockHeigh(): number {
 	const file = JSON.parse(fs.readFileSync(fileToCheck).toString());
 	return file.blockHeight + 1;
 }
+
+export function getLastTimestamp(): number {
```

Comment:
When I run the first test after install with no `daily_output.json` file, the tests all pass. But after the first test, this `daily_output.json` file is produced:

```
!{{ NOT A VALID JSON "": false
```

Then this tests fail to complete on this function with this stack trace:

```shell
  1) Uncaught error outside test suite:
     Uncaught SyntaxError: Unexpected token ! in JSON at position 0
      at JSON.parse (<anonymous>)
      at getLastTimestamp (src/common.ts:2:4051)
      at new DailyOutput (src/daily_output.ts:4:346)
      at Suite.<anonymous> (src/daily_output.test.ts:75:17)
      at Object.<anonymous> (src/daily_output.test.ts:74:1)
      at Module._compile (internal/modules/cjs/loader.js:1068:30)
      at Module.replacementCompile (.yarn/cache/append-transform-npm-2.0.0-99bd7d69ed-f26f393bf7.zip/node_modules/append-transform/index.js:60:13)
      at Module.m._compile (.yarn/__virtual__/ts-node-virtual-9002d3d23b/0/cache/ts-node-npm-10.7.0-ef39b1d45e-2a379e43f7.zip/node_modules/ts-node/src/index.ts:1455:23)
```

Reply:
Remove the file after the tests run: 7dc58d42a626c9e85010858e7552a938533d7ac7